source/server/listener_impl.cc

```diff
@@ -407,18 +408,18 @@ ListenerImpl::ListenerImpl(ListenerImpl& origin,
       quic_stat_names_(parent_.quicStatNames()) {
   buildAccessLog();
   auto socket_type = Network::Utility::protobufAddressSocketType(config.address());
-  // buildUdpListenerFactory() must come before buildListenSocketOptions() because the UDP
-  // listener factory can provide additional options.
-  buildUdpListenerFactory(socket_type, concurrency);
```

Comment:
Just so I understand this is removed because this is the in-place update path, which was never supported before for UDP, and now that it is, we use the existing factory or just don't need a factory? Is that right?